README.md

```diff
@@ -145,6 +145,11 @@ just hang out ☕.
     <td>Text-to-Image</td>
     <td><a href="https://huggingface.co/docs/diffusers/api/pipelines/if">if</a></td>
       <td><a href="https://huggingface.co/DeepFloyd/IF-I-XL-v1.0"> DeepFloyd/IF-I-XL-v1.0 </a></td>
+  </tr>
+    <tr>
+    <td>Text-to-Image</td>
+    <td><a href="https://huggingface.co/docs/diffusers/api/pipelines/kandinsky">Kandinsky</a></td>
+      <td><a href="https://huggingface.co/kandinsky-community/kandinsky-2-2-decoder"> kandinsky-community/kandinsky-2-2-decoder </a></td>
   </tr>
```

Comment:
Should we not include both Kandinsky 1 and 2?

Reply:
but even SD doesn't get two checkpoints there😅

Reply:
We should probably at least include the most impactful ones:

* SD 1.4
* SD 1.5
* SD 2.1
* SD XL 0.9
* SD XL 1.0 (when it comes out)

@stevhliu @patrickvonplaten WDYT?

Reply:
Think we can add a new PR to include SD XL, but think this PR is good as is